modules/open-close.html

```diff
@@ -0,0 +1,307 @@
+<!DOCTYPE html>
```

Comment:
At some point these probably DO need to revert to .php files with the header restored:
```
<?php
include('../wp-blog-header.php');

global $user_identity;
if($user_identity){
	echo $user_identity;
}
else{
	//die('Sorry, you must be <a href="'. get_bloginfo('home') . '/wp-login.php?redirect_to=' . $_SERVER['PHP_SELF'] . '">logged in</a> to view this page.');
}
//echo do_shortcode( '[contact-form-7 id="1234" title="Contact form 1"]' );
?>
```

Seems like we can push ahead for now, but maybe put the PHP headers back in as a comment with a note to modify these files back at a later point after development is closer to the end?

Reply:
<!--
TODO: uncomment this php header and restore file type to .php before making live
<?php
include('../wp-blog-header.php');

global $user_identity;
if($user_identity){
	echo $user_identity;
}
else{
	//die('Sorry, you must be <a href="'. get_bloginfo('home') . '/wp-login.php?redirect_to=' . $_SERVER['PHP_SELF'] . '">logged in</a> to view this page.');
}
//echo do_shortcode( '[contact-form-7 id="1234" title="Contact form 1"]' );
?>
//-->

Reply:
I think that content is being pulled from WordPress which seems unnecessary!  I like the idea of just putting this into HTML and CSS with static content, but we should look into where that content originates and if WordPress is needed.  I might be missing something, but I just see dropdown menus with links.  This doesn't appear as if it needs to be pulled from WordPress:
![image](https://github.com/user-attachments/assets/72207145-1f08-4ebe-9e30-f6285bf8677f)